airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -477,6 +481,27 @@ def _make_safe_pod_id(safe_dag_id, safe_task_id, safe_uuid):
 
         return safe_pod_id
 
+    @staticmethod
+    def _make_safe_label_value(string):
+        """
+        Valid label values must be 63 characters or less and must be empty or begin and
+        end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_),
+        dots (.), and alphanumerics between.
+
+        If the label value is then greater than 63 chars once made safe, or differs in any
+        way from the original value sent to this function, then we need to truncate to
+        53chars, and append it with a unique hash.
```

Comment:
`53chars` is a typo?

Reply:
No that's correct.  If we are hashing the string after it's been modified in any way, we need to truncate the result from `AirflowKubernetesScheduler._make_safe_label_value()` to a maximum of 53chars, as we are adding a 9 char has to the end.  If we were to truncate to 63chars, when the has is added, it would then make the string 73chars max length, which would be too long for the label value length limit.